airflow/providers/amazon/aws/links/glue.py

```diff
@@ -0,0 +1,29 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from airflow.providers.amazon.aws.links.base_aws import BASE_AWS_CONSOLE_LINK, BaseAwsLink
+
+
+class GlueJobLogsLink(BaseAwsLink):
+    """Helper class for constructing AWS Glue Job Logs Link"""
+
+    name = "AWS Glue Job Logs"
+    key = "glue_job_logs"
```

Comment:
As far as i understand you want to add link to `AWS Glue Studio ❯ Monitoring ❯ Job run` not link to Cloudwatch logs

So better named it appropriate like (it suggested renaming, final choice by you): 
GlueJobLogsLink > GlueJobRunDetailsLink
"AWS Glue Job Logs" > "AWS Glue Job Run Details"
"glue_job_logs" > "glue_job_run_details"

Reply:
It might also make sense to add the CloudWatch logs link?
Maybe I will add them on a new PR

Reply:
Yep, why not? We already add some similar for BatchOperator.

I think for Glue it might required create a bit different link, e.g. Batch use link to a single LogStream, but Glue create multiple different log stream during execution so potentially we need to provide link to filter LogStream by specific Job ID